test/unit/assets/utils.ts

```diff
@@ -65,8 +64,12 @@ export function cleanup() {
   });
 }
 
+export function tick(time) {
+  (window as any).timer.tick(time);
+}
+
 declare var Simulator: any;
-export function simulate(el: HTMLElement, option?: object): Promise<void> {
+export function simulate(el: HTMLElement, option?: object, time: number = 15000): Promise<void> {
```

Comment:
Isn't it working synchronously?

Reply:
Actually it is.
But I just let this function asynchronous as hammer-simulator calls `done()` after panning is finished.
I think it can be used more practically when we return Promise in this function. Because we can assume panning is finished when the promise is resolved.

Reply:
Ok, I got it.